### DISCUSSION TITLE
Error 403: rate_limit_exceeded

### DISCUSSION THREAD

**Discussion by Adambean** [kewisch/gdata-provider#296] [Support] [answered]
I'm sure the torrent of reviews about "Error 403: rate_limit_exceeded" when signing in (as of TB 91's release) has already been seen:
https://addons.thunderbird.net/en-us/thunderbird/addon/provider-for-google-calendar/reviews/

I've not seen a support thread here though so just posting in case the reviews have not been seen. Essentially due to thousands of people needing to authenticate with Google again in this add-on Google is blocking successful authentications due to API rate limits being exceeded.

Exact error message:

> **Error 403: rate_limit_exceeded**
> 
> This app has reached its sign-in rate limit for now.
> 
> Google limits how quickly an app can get new users. You can try signing in again later or ask the developer (provider-for-google-calendar@googlegroups.com) to increase this app's sign-in rate limit.
> 
> If you are the developer for this app, you can request a sign-in rate limit increase.

Currently this add-on is completely non-functional to many users (probably way more than have bothered to leave a review) due to this.

Thanks for the add-on though, it's been working very splendidly for many years.

-- Adam

**Comment by DanielBarreleiro** [comment, marked as answer]
This is known, there is an open issue for this:
#288

**Reply by Adambean** [reply]
Thanks for your quick reply @DanielBarreleiro 👍 I hadn't checked the issues section (the reviews area sent me here)
I'll upvote that issue instead.
